cmd/vela-s3-cache/repo.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Repo represents the available settings for repository.
+type Repo struct {
+	Owner       string
+	Name        string
+	Branch      string
+	BuildBranch string
+}
+
+// Validate verifies the repo configuration.
+func (r *Repo) Validate() error {
```

Comment:
minimum information required.. i settled for `Owner` and `Name` but can definitely see others here as well.